src/Components/Health/Checker/PerformanceChecker/MysqlSettingsChecker.php

```diff
@@ -62,6 +65,22 @@ private function checkSqlMode(HealthCollection $collection): void
         }
     }
 
+    private function checkTimeZone(HealthCollection $collection): void
+    {
+        $timeZone = $this->connection->fetchOne('SELECT @@time_zone');
+        if (\is_string($timeZone) && $timeZone !== self::MYSQL_TIME_ZONE) {
+            $collection->add(
+                SettingsResult::error(
+                    'sql_time_zone',
+                    'MySQL value time_zone',
+                    $timeZone,
+                    self::MYSQL_TIME_ZONE,
+                    self::DOCUMENTATION_URL,
+                ),
+            );
```

Comment:
I guess this should be a warning. Shopware does set the Timezone by default, so it is not critical. Additionally there are values like `SYSTEM` which are not the same as `+00:00` but might still be correct. Since there are multiple ways to configure the timezone we shouldn't be too strict.

```suggestion
            $collection->add(
                SettingsResult::warning(
                    'sql_time_zone',
                    'MySQL value time_zone',
                    $timeZone,
                    self::MYSQL_TIME_ZONE,
                    self::DOCUMENTATION_URL,
                ),
            );
```